src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -27,9 +29,13 @@
     private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
 
     @Test
-    public void execute_validIndexUnfilteredList_success() {
-        Person personToDelete = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
-        DeleteCommand deleteCommand = new DeleteCommand(INDEX_FIRST_PERSON);
+    public void execute_validCLientIdUnfilteredList_success() {
```

Comment:
Perhaps the method name should be this
```suggestion
    public void execute_validClientIdUnfilteredList_success() {
```